packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ReactElement} from 'react'
+
+interface CommerceAPIProviderProps {
+    clientId: string
+    organizationId: string
+    siteId: string
+    shortCode: string
+    proxy: string
+    locale: string
+    currency: string
+}
+
+const CommerceAPIProvider = (props: CommerceAPIProviderProps): ReactElement => {
+    return <></>
+}
+
+export default CommerceAPIProvider
```

Comment:
With the library being named `commerce-sdk-react` with the expectation that we may put other things commerce sdk'y in here. Do you think we need a provider folder?

Another note. Are you planning on making use of index files at all? Might clean up some import statements potentially.

Reply:
let's create the folder when we have more than 1 provider

Reply:
It might be too late at that point! If people import using the full file path, then changing `provider.tsx` to `provider/index.tsx` is a breaking change.